File: timing.go
//https://gobyexample.com/time
//https://pkg.go.dev/time

package main

import (
    "fmt"
    "time"
    "strconv"
)

func main() {
    start := time.Now()
    time.Sleep(10 * time.Second)
    t := time.Now()
    elapsed := t.Sub(start)
    elapsed_string := strconv.FormatFloat(elapsed.Seconds(),'f',6,64)
    fmt.Printf("Time taken %s seconds\n", elapsed_string)
}

File: timing.py
import time 

t0 = time.time_ns()
time.sleep(10)
t1 = time.time_ns()
time_diff = t1-t0
time_diff_s = time_diff / (10 ** 9) # convert to floating-point seconds
print("Time taken {0:.6f} seconds".format(time_diff_s))